docs/secpoll.zone

```diff
@@ -192,6 +192,7 @@ recursor-4.2.0-alpha1.security-status                   60 IN TXT "1 OK"
 recursor-4.2.0-beta1.security-status                    60 IN TXT "1 OK"
```

Comment:
Please change the pre-releases to "2 Unsupported pre-release, no known vulnerabilities"